R/reactives.R

```diff
@@ -1583,6 +1708,20 @@ maskReactiveContext <- function(expr) {
 #' @param ignoreNULL Whether the action should be triggered (or value
 #'   calculated, in the case of \code{eventReactive}) when the input is
 #'   \code{NULL}. See Details.
+#' @param skipFirst Whether the action should be run the first time the
```

Comment:
Ah, perfect! Thanks @jcheng5! That is definitely the clearest way to explain this that any of us has come up with so far! This: "when the observer is first created/initialized, skip/ignore the handler whether it is otherwise supposed to run or not" makes the purpose of `ignoreInit` so much clearer even in my own head!